pkg/syncthing/syncthing.go

```diff
@@ -385,19 +327,21 @@ func (s *Syncthing) WaitForCompletion(ctx context.Context, wg *sync.WaitGroup, d
 
 // Restart restarts the syncthing process
 func (s *Syncthing) Restart(ctx context.Context, wg *sync.WaitGroup) error {
-	log.Infof("restarting synchting...")
-	_, err := s.APICall("rest/system/restart", "POST", 200, nil, true, nil)
-	return err
-}
-
-// Stop halts the background process and cleans up.
-func (s *Syncthing) Stop() error {
-	pidPath := filepath.Join(s.Home, syncthingPidFile)
+	log.Infof("restarting syncthing...")
+	status := s.app.Stop(syncthing.ExitSuccess)
+	if status == syncthing.ExitError {
+		log.Infof("failed to restart syncthing")
+		return fmt.Errorf("internal server error")
+	}
 
-	if err := s.cleanupDaemon(pidPath); err != nil {
+	s.app.Wait()
+	log.Infof("syncthing stopped")
+	if err := s.initSyncthingApp(); err != nil {
```

Comment:
why are we restarting here?

Reply:
it was already like that before. I believe is when the change in `sendtype` happens.